package capkproto;

import "capk_globals.proto";

message venue_configuration {
    optional string     venue_id = 1;
    optional string     mic_name = 2;
    optional string     order_interface_addr = 3; 
    optional string     order_ping_addr = 4; 
    optional string     market_data_broadcast_addr = 5;

}

message configuration {
    optional string     trade_serialization_addr = 1;
    optional string     recovery_listener_addr = 2;
    repeated venue_configuration configs = 3;
}
